l10n_fr_das2/README.rst

```diff
@@ -59,7 +62,7 @@ You should also have a look in the chatter: you may have a message that warn you
 
 Once your declaration is OK, click on the button *Generate File* and download the generated file.
 
-Connect to `https://teletd.impots.gouv.fr/teletd/connexionEFI.do <https://teletd.impots.gouv.fr/teletd/connexionEFI.do>`_, select *Transmission par internet des fichiers TD/bilatéral* and type your login and password (the credentials are specific to this website; their are not the same as on *impots.gouv.fr*). Then follow the online declaration process and upload the file generated by Odoo.
+Connect to DAS2_declaration_, select *Transmission par internet des fichiers TD/bilatéral* and type your login and password (the credentials are specific to this website; their are not the same as on *impots.gouv.fr*). Then follow the online declaration process and upload the file generated by Odoo.
```

Comment:
@alexis-via I made this fix to make travis green again after commits of l10n_fr_das2.
Please if you could only merge when it's green it could be easier for all of us :-)

But maybe you couldn't ? ;-)